chapters/publication.tex

```diff
@@ -99,91 +174,257 @@ \section{Collaborating on academic writing}
+(If you do not provide a license, nobody can use your code!)
+
+Make sure the code functions identically on a fresh install of your chosen software.
+A new user should have no problem getting the code to execute perfectly.
+In either a scripts folder or in the root directory,
+include a master script (dofile or Rscript for example).
+The master script should allow the reviewer
+to change one line of code setting the directory path.
 Then, running the master script should run the entire project
 and re-create all the raw outputs exactly as supplied.
-Check that all your code will run completely on a new computer.
-This means installing any required user-written commands in the master script
+Indicate the filename and line to change.
+Check that all your code will run completely on a new computer:
+Install any required user-written commands in the master script
 (for example, in Stata using \texttt{ssc install} or \texttt{net install}
 and in R include code for installing packages,
-including installing the appropriate version of the package if necessary).
-Make sure settings like \texttt{version}, \texttt{matsize}, and \texttt{varabbrev} are set.\sidenote{In Stata, \texttt{ietoolkit}'s \texttt{ieboilstart} command will do this for you.}
-All outputs should clearly correspond by name to an exhibit in the paper, and vice versa.
+including selecting a specific version of the package if necessary).
+In many cases you can even directly provide the underlying code
+for any user-installed packages that are needed to ensure forward-compatibility.
+Make sure system settings like \texttt{version}, \texttt{matsize}, and texttt{varabbrev} are set.
+
+Finally, make sure that the code and its inputs and outputs are clearly identified.
+A new user should, for example, be able to easily identify and remove
+any files created by the code so that they can be recreated quickly.
+They should also be able to quickly map all the outputs of the code
+to the locations where they are placed in the associated published material,
+such as ensuring that the raw components of figures or tables are clearly identified.
+For example, outputs should clearly correspond by name to an exhibit in the paper, and vice versa.
+(Supplying a compiling \LaTeX\ document can support this.)
+Code and outputs which are not used should be removed.
+
+\subsection{Releasing a replication package}
+
+If you are at this stage,
+all you need to do is find a place to publish your materials.
+This is slightly easier said than done,
+as there are a few variables to take into consideration
+and no global consensus on the best solution.
+The technologies available are likely to change dramatically
+over the next few years;
+the specific solutions we mention here highlight some current approaches
+as well as their strengths and weaknesses.
+GitHub provides one solution.
+Making your GitHub repository public
+is completely free for finalized projects.
```

Comment:
But not for ongoing projects?

Reply:
Fixed in ffed54381a980a7a435af263d597e35ac9637c46